scripts/irods/test/test_ipasswd.py

```diff
@@ -224,11 +224,6 @@ def test_ipasswd_rodsuser_should_fail(self):
         str1 = "CAT_INVALID_AUTHENTICATION: failed to perform request"
         self.assertIn(str1, stderr, "ipasswd: Expected stderr: \"...{0}...\", got: \"{1}\"".format(str1, stderr))
 
-        # and...
-
-        str1 = "failed with error -826000 CAT_INVALID_AUTHENTICATION"
-        self.assertIn(str1, stderr, "ipasswd: Expected stderr: \"...{0}...\", got: \"{1}\"".format(str1, stderr))
-
```

Comment:
Has the output changed for ipasswd?

Reply:
See https://github.com/irods/irods/pull/7149#discussion_r1270750865